storage/src/main/java/org/apache/uniffle/storage/factory/ShuffleHandlerFactory.java

```diff
@@ -136,7 +138,11 @@ private ClientReadHandler getHdfsClientReadHandler(CreateShuffleReadHandlerReque
         request.getExpectBlockIds(),
         request.getProcessBlockIds(),
         request.getStorageBasePath(),
-        request.getHadoopConf());
+        request.getHadoopConf(),
+        request.getDistributionType(),
+        request.getStartMapIndex(),
```

Comment:
Do ClientReadHandler need startMapIndex and EndMapIndex?
I think it's a Spark AQE concept, I prefer not exposing them?
What do you think?

Reply:
+1.

Reply:
Done